test/integration/targets/git/vars/main.yml

```diff
@@ -1,5 +1,8 @@
 ---
 
+source_dir:  "{{ output_dir }}/source"
```

Comment:
`msg` is used twice and since its quite a long line, it has been added as a var.
other vars have not been taken care of.